platform_versioned_docs/version-23.4.0/enterprise/prerequisites/azure.mdx

```diff
@@ -7,182 +7,310 @@ tags: [azure, prerequisites, configuration]
-An [Azure Kubernetes Service (AKS)](https://docs.microsoft.com/en-us/azure/aks/kubernetes-walkthrough-portal) cluster is required to deploy Seqera Enterprise via Kubernetes. See the [AKS documentation](https://learn.microsoft.com/en-us/azure/architecture/aws-professional/eks-to-aks/) to provision your own cluster.
+An [Azure Kubernetes Service (AKS)][aks-walkthrough] cluster is required to deploy Seqera Enterprise via Kubernetes.
 
-To customize your cluster's Ingress Controller to support HTTPS redirects and TLS certificates, see [these instructions](https://azure.github.io/application-gateway-kubernetes-ingress/).
+To customize your cluster's Ingress Controller to support HTTPS redirects and TLS certificates, see [Application Gateway Ingress Controller][application-gateway-ingress].
 
-### Seqera container images
+## Azure setup
 
-Seqera Platform Enterprise is distributed as a collection of Docker containers available through the Seqera
-container registry ([cr.seqera.io](https://cr.seqera.io)). Contact [support](https://support.seqera.io) to get your container access credentials. Once you've received your credentials, retrieve the Seqera container images:
+Set up commonly used Azure services for Seqera deployment.
 
-1. Retrieve the **username** and **password** you received from Seqera support.
-2. Run the following Docker command to authenticate to the registry (using the `username` and `password` values copied in step 1):
+### Azure resource group
 
-   ```bash
-   docker login -u '/\<USERNAME\>/' -p '/\PASSWORD\>/' cr.seqera.io
-   ```
+Create a resource group:
+- [Via the Azure portal][azure-rg-portal]
+- [Via the Azure CLI][azure-rg-cli]
 
-3. Pull the Seqera container images with the following commands:
+<details>
+   <summary>Create a resource group via Azure portal</summary>
 
-   ```bash
-   docker pull cr.seqera.io/private/nf-tower-enterprise/backend:v23.4.3
+  1. Sign in to the [Azure portal](https://portal.azure.com).
 
-   docker pull cr.seqera.io/private/nf-tower-enterprise/frontend:v23.4.3
-   ```
+  1. Select **Resource groups**.
 
-`cr.seqera.io` is the default Seqera container image registry from version 22.4. Use of the AWS, Azure, and Google Cloud image registries in existing installations is still supported but is deprecated for **new installations** from June 2023. See [here](../advanced-topics/seqera-container-images.mdx) for steps to use the Seqera private Azure registry.
+  1. Select **Add**.
 
-## Azure setup
+  1. Enter the following values:
 
-This section provides step-by-step instructions for some commonly used Azure services for Seqera deployment. See the [Azure documentation](https://docs.microsoft.com/en-us/azure/?product=popular) for up-to-date instructions and contact [Azure support](https://support.microsoft.com/en-us/topic/contact-microsoft-azure-support-2315e669-8b1f-493b-5fb1-d88a8736ffe4) if you have any issues with provisioning Azure resources.
+    - **Subscription**: Select your Azure subscription.
```

Comment:
The indention for this is off somehow. The list breaks & resets at 1.

<img width="272" alt="Screenshot 2024-05-17 at 1 14 20 PM" src="https://github.com/seqeralabs/docs/assets/141646877/370529b4-4f5e-4e37-9a14-6d8311109c98">